src/utils/check-pr/check-pr.py

```diff
@@ -277,7 +277,16 @@ def run(self, *, merge_on_success: bool = False) -> None:
 
 
 def main() -> None:
-    default_instance = "pr-check-%s" % uuid.uuid4().hex
+    # Get a name that can be added to the resource group name
+    # to make it easy to identify the owner
+    cmd = ["az", "ad", "signed-in-user", "show", "--query", "mailNickname"]
+    name=subprocess.run(cmd, capture_output=True, encoding='UTF-8')
```

Comment:
You should use `subprocess.check_output` to validate the command completed successfully.  Additionally, you can get the output without quotes by using the flags: `-o tsv`